README.md

```diff
@@ -72,143 +75,29 @@ Directives from this repository are automatically tested with the following brow
 
 Modern mobile browsers should work without problems.
 
-**IE 8 is not officially supported at the moment**. This project is run by volunteers and with the current number of commiters
-we are not in the position to guarantee IE8 support. If you need support for IE8 we would welcome a contributor who would like to take care about IE8.
-Alternatively you could sponsor this project to guarantee IE8 support.
-
-We believe that most of the directives would work OK after:
-* including relevant shims (for ES5 we recommend https://github.com/kriskowal/es5-shim)
-* taking care of the steps described in http://docs.angularjs.org/guide/ie
-
-We are simply not regularly testing against IE8.
-
-## Project philosophy
-
-### Native, lightweight directives
-
-We are aiming at providing a set of AngularJS directives based on Bootstrap's markup and CSS. The goal is to provide **native AngularJS directives** without any dependency on jQuery or Bootstrap's JavaScript.
-It is often better to rewrite an existing JavaScript code and create a new, pure AngularJS directive. Most of the time the resulting directive is smaller as compared to the original JavaScript code size and better integrated into the AngularJS ecosystem.
-
-### Customizability
-
-All the directives in this repository should have their markup externalized as templates (loaded via `templateUrl`). In practice it means that you can **customize directive's markup at will**. One could even imagine providing a non-Bootstrap version of the templates!
+> **Internet Explorer 8 support**
 
-### Take what you need and not more
+> Since AngularJS 1.3, the Angular team has stopped supporting Internet Explorer 8. From UI Bootstrap version `0.13.0` on, we will update the AngularJS dependency to version 1.3 and by doing so **we will drop all Internet Explorer 8 support**. (New) issues regarding Internet Explorer 8 will be closed.
```

Comment:
We didnt really discuss that last meeting. I thought we were pushing forward. Since we never really did extensive tests on IE8 in the first place (based on notes in the old README) I figured this would make sense.